org.mixedrealitytoolkit.uxcomponents/Tests/Runtime/SeeItSayItCanvasTests.cs

```diff
@@ -86,6 +86,7 @@ public IEnumerator TestLabelEnabledOnHover()
             yield return hand.Show(Vector3.one);
             yield return RuntimeTestUtilities.WaitForUpdates();
             yield return hand.MoveTo(testButton.transform.position);
+
```

Comment:
This file only seems to have newline changes, intentional?

Reply:
Improves readability :)